lib/descriptor/type/addr.js

```diff
@@ -0,0 +1,127 @@
+/*!
+ * addr.js - address descriptor object for bcoin
+ * Copyright (c) 2023, the bcoin developers (MIT License).
+ * https://github.com/bcoin-org/bcoin
+ */
+
+'use strict';
+
+const AbstractDescriptor = require('../abstractdescriptor');
+const assert = require('bsert');
+const common = require('../common');
+const {isType, strip, checkChecksum, scriptContext, types} = common;
+const Address = require('../../primitives/address');
+const Network = require('../../protocol/network');
+
+/**
+ * AddressDescriptor
+ * Represents the output script produced by the address in the descriptor.
+ * @see https://github.com/bitcoin/bips/blob/master/bip-0385.mediawiki#addr
+ * @property {String} type
+ * @property {Address} address
+ * @property {Network} network
+ * @extends Descriptor
```

Comment:
```suggestion
 * @extends AbstractDescriptor
```

here and in all type files

Reply:
resolved